doc/source/ray-observability/user-guides/debug-apps/general-debugging.rst

```diff
@@ -154,107 +136,55 @@ the newest version.
   def f():
       return 2
 
-  print(ray.get(f.remote()))  # This should be 2.
+  print(ray.get(f.remote()))  # This should print 2.
 
 .. testoutput::
 
   2
 
-However, the following are cases where modifying the remote function will
-not update Ray to the new version (at least without stopping and restarting
-Ray).
-
-- **The function is imported from an external file:** In this case,
-  ``f`` is defined in some external file ``file.py``. If you ``import file``,
-  change the definition of ``f`` in ``file.py``, then re-``import file``,
-  the function ``f`` will not be updated.
-
-  This is because the second import gets ignored as a no-op, so ``f`` is
-  still defined by the first import.
-
-  A solution to this problem is to use ``reload(file)`` instead of a second
-  ``import file``. Reloading causes the new definition of ``f`` to be
-  re-executed, and exports it to the other machines. Note that in Python 3, you
-  need to do ``from importlib import reload``.
-
-- **The function relies on a helper function from an external file:**
-  In this case, ``f`` can be defined within your Ray application, but relies
-  on a helper function ``h`` defined in some external file ``file.py``. If the
-  definition of ``h`` gets changed in ``file.py``, redefining ``f`` will not
-  update Ray to use the new version of ``h``.
-
-  This is because when ``f`` first gets defined, its definition is shipped to
-  all of the Worker processes, and is unpickled. During unpickling, ``file.py`` gets
-  imported in the Workers. Then when ``f`` gets redefined, its definition is
-  again shipped and unpickled in all of the Workers. But since ``file.py``
-  has been imported in the Workers already, it is treated as a second import
-  and is ignored as a no-op.
-
-  Unfortunately, reloading on the Driver does not update ``h``, as the reload
-  needs to happen on the worker.
-
-  A solution to this problem is to redefine ``f`` to reload ``file.py`` before
-  it calls ``h``. For example, if inside ``file.py`` you have
-
-  .. testcode::
+However, there are cases where modifying a remote function doesn't take effect without restarting the cluster:
 
-    def h():
-        return 1
+— **Imported function issue:** If ``f`` is defined in an external file (e.g., ``file.py``), and you modify its definition, re-importing the file may be ignored because Python treats the subsequent import as a no-op. A solution is to use ``from importlib import reload; reload(file)`` instead of a second import.
```

Comment:
is there a way to disable this behavior? I've had this issue when actively developing on a running cluster and it would be useful to disable this optimization.